src/include/kdbopmphm.h

```diff
@@ -22,17 +22,25 @@
  * opmphmRatio should never be less than 1.
  *
  */
-#define OPMPHMTUPLE 7
+#define OPMPHMTUPLE 5
```

Comment:
Could you please give rationales for how you came up with the number. The description says what it does, but not why you chose the specific number.